dotcom-rendering/src/experiments/tests/deeply-read-right-column.ts

```diff
@@ -0,0 +1,35 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const deeplyReadRightColumn: ABTest = {
+	id: 'DeeplyReadRightColumn',
+	author: '@dotcom-platform',
+	start: '2024-04-30',
+	expiry: '2024-07-31',
+	audience: 0 / 100,
+	audienceOffset: 0 / 100,
+	audienceCriteria: '',
+	successMeasure: 'Improved CTR',
+	description:
+		'Test the impact of adding deeply read component to the right column.',
+	variants: [
+		{
+			id: 'control',
```

Comment:
If you had a real control group, it would need to not change the API URL, so I'd suggest naming this **most-read-only**